connect_labs/mcp/tools/synthetic.py

```
@@ -1639,14 +1685,15 @@ def synthetic_clone_profile(user, *, spec_yaml: str) -> dict[str, Any]:
         "additionalProperties": False,
     },
     is_write=True,
+    wants_progress=True,
 )
-def synthetic_clone_generate(user, *, spec_yaml: str, fresh: bool = False) -> dict[str, Any]:
+def synthetic_clone_generate(user, *, spec_yaml: str, fresh: bool = False, progress=NULL_PROGRESS) -> dict[str, Any]:
     try:
         spec = CohortSpec.from_yaml(spec_yaml)
     except ValueError as exc:
         raise MCPToolError("INVALID_SCHEMA", str(exc))
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)